environment.yml

```diff
@@ -15,16 +16,30 @@ dependencies:
   - numpy
   - pandas
   - pip
-  - pyerfa
+  - pyerfa=1.7.3
   - pytables
   - pytest
   - pytest-cov
   - python=3.9
   - pyyaml
+  - scikit-hep>3.1,<3.3
   - scikit-learn
   - scipy
   - tqdm
   - twine
   - wheel
+  # for zfit
+  - iminuit
+  - nlopt
+  - numdifftools
+  - tensorflow
+  - tensorflow-probability
+  - texttable
+  - ordered-set
+  - tabulate
+  - uproot
+  - dotmap
   - pip:
+    - zfit>0.5,<0.7
```

Comment:
```suggestion
    - zfit>0.6,<0.8
```

Reply:
The new version is gonna be 0.7.x

Reply:
Thanks. I was hoping to do that change in a subsequent MR but this one ended up being far less trivial than I thought. No harm to do it now and here.